spec/services/ccms/requestors/case_add_requestor_spec.rb

```diff
@@ -205,6 +206,21 @@ module Requestors
             end
           end
         end
+
+        context "when multiple opponents are added" do
+          let(:opponent_one) { create(:opponent, first_name: "Joffrey", last_name: "Test-Opponent") }
+          let(:opponent_two) { create(:opponent, first_name: "Sansa", last_name: "Opponent-Test") }
+          let(:opponents) { [opponent_one, opponent_two] }
+
+          before do
+            allow(opponent_one).to receive(:generate_ccms_opponent_id).and_return("123456")
+            allow(opponent_two).to receive(:generate_ccms_opponent_id).and_return("654321")
+          end
+
+          it "generates expected xml with multiple opponents" do
+            expect(expected_xml).to match(/Sansa Opponent-Test/)
```

Comment:
Is this correct? (that only the Sansa Opponent-Test is included  in the xml and not Joffrey Test-Opponent)? Could the description be clearer? (e.g. it "generates expected xml with first opponent")

Reply:
Good catch, it was my initial test that the _new_ value was at least found.  Extended to ensure it matches both :+1: